### ISSUE TITLE
File size is big

### ISSUE THREAD

**Issue by ggepenyan** [highcharts/highcharts-react#273]
I have installed the package with **npm install highcharts highcharts-react-official** this command and file sizes after the build process were increased by **90.33 kb**. What do you think about it and are there any suggestions? Thank you 🙂

**Comment by ggepenyan** [comment]
I'm using:
"react": "^16.9.0",
"react-scripts": "^3.3.0",
"highcharts": "^8.2.2",
"highcharts-react-official": "^3.0.0"

**Comment by Basts** [comment]
Hello @ggepenyan,

Thank you for contacting us.
I did the following steps to reproduce your case:

1. Created a new react-app folder by using `npx create-react-app`,
2. Run the build command - `yarn build`,
3. Checked the build file size which is for the pure/new react-app equal 557 kb,
4. Installed the Highcharts React Wrapper by using `npm install highcharts highcharts-react-official`,
5. Checked the build file size again and it is still equal 557 kb,

Have you done anything else? For now, I am not able to reproduce it.

**Comment by alexandermirzoyan** [comment]
Hi everyone, I have the same problem too. Build file size has increased by 90.33KB after implementing a component from highcarts's library.

**Comment by Basts** [comment]
@alexandermirzoyan @ggepenyan does the size has increased after importing the Highcharts library and the wrapper into your projects?

**Comment by ggepenyan** [comment]
@Basts In my case yes. It increased after I did imports:
```
import Highcharts from 'highcharts';
import HighchartsReact from 'highcharts-react-official';
```
And I can see your packages' sizes on bundlephobia:
Highcharts - 89.4kb - https://bundlephobia.com/result?p=highcharts@8.2.2
Highcharts react official - 1.1kb - https://bundlephobia.com/result?p=highcharts-react-official@3.0.0
Sum is greater than 90kb

**Comment by Basts** [comment]
@ggepenyan,

Thank you for your explanation. 

As you can notice you have imported the whole Highcharts library into your project, which size is 89.4kb. Highcharts React Official is just a wrapper that enables to use the Highcharts library in the React envoirment.

If you want to decrease the size of the library you can customize the Highcharts files which will be imported into your project: https://www.highcharts.com/docs/getting-started/how-to-create-custom-highcharts-files

To sum up - this problem is not related to working the Highcharts React wrapper.
